tests/test_suite.py

```diff
@@ -217,7 +217,7 @@ def monitorOutputFile(finalMessage):
 thread.start()
 
 webotsArguments = '--mode=fast --no-rendering --stdout --stderr --minimize --batch'
-webotsArgumentsRendering = '--mode=realtime --stdout --stderr --batch'
+webotsArgumentsRendering = '--mode=fast --stdout --stderr --batch'
```

Comment:
That would seem more logical to me:
```suggestion
webotsArguments = '--mode=fast --stdout --stderr --batch'
webotsArgumentsNoRendering = webotsArguments + ' --no-rendering --minimize'
```
And then we should adapt the code to `webotsArgumentsNoRendering`.